src/server.c

```diff
@@ -5742,8 +5741,8 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "mem_cluster_links:%zu\r\n", mh->cluster_links,
             "mem_aof_buffer:%zu\r\n", mh->aof_buffer,
             "mem_allocator:%s\r\n", ZMALLOC_LIB,
-            "mem_databases_overhead_main:%zu\r\n", databases_overhead_main,
-            "mem_databases_overhead_expires:%zu\r\n", databases_overhead_expires,
+            "mem_db_bucket_overhead_ht0:%zu\r\n", server.db_bucket_overhead_ht0 * sizeof(dictEntry*),
+            "mem_db_bucket_overhead_ht1:%zu\r\n", server.db_bucket_overhead_ht1 * sizeof(dictEntry*),
```

Comment:
i think that intead of `mem_db_bucket_overhead_ht0` and `mem_db_bucket_overhead_ht1`, which are low level details.
we should have `mem_db_hashtable_overhead_total` and `mem_db_hashtable_overhead_rehashing`
i.e. the `total` should be ht0+ht1, and the `rehashing` is just ht1.

Reply:
actually, looking at my https://github.com/redis/redis/pull/12913#pullrequestreview-1827646066.
the `rehashing` metric is not `ht1`, it's `ht0` of dicts that also have `ht1` allocated (dicts that are undergoing rehashing).
so the raw counters should be different than what you implemented.

Reply:
So once `ht1` is allocated, it is considered as the "main" hash table, and `ht0` becomes rehashing overhead?

In this case,
`mem_db_hashtable_overhead_total` == all `ht1` plus all `ht0` in dicts that are not rehashing
`mem_db_hashtable_overhead_rehashing` == all `ht0` in dicts that are rehashing

Am I right?